### DISCUSSION TITLE
How to display a flash message in a component the Directus way?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#14503] [Archive]
In my component, I'd like to show a message in the way Directus does, for instance, showing `Item saved` for 2 seconds after saving.
Is there a ready made function I can call within the component or should I rebuild that with bare Vue?

**Comment by rijkvanzanten** [comment]
You can use the notifications store for that:

```js
const { useNotificationsStore } = useStores();
const notificationsStore = useNotificationsStore()'

notificationsStore.add({
  title: 'Hello World!'
});
```

The full type for the notifications store add method can be found here https://github.com/directus/directus/blob/main/app/src/types/notifications.ts
